### PR TITLE
Initial Layout Page 

### PR DESCRIPTION
**Pull request by ainouzgali**
### What change does this PR introduce?

Rough start of layout pages - list of layouts + edit layout

### Other information (Screenshots)
Screen Shot 2023-01-10 at 13 19 44
Screen Shot 2023-01-10 at 13 19 59

### CODE REVIEW

File: apps/web/src/pages/brand/BrandPage.tsx [hunk 1]
```diff
@@ -7,8 +7,10 @@ import PageContainer from '../../components/layout/components/PageContainer';
 import { Tabs } from '../../design-system';
 import { AuthContext } from '../../store/authContext';
 import { BrandingForm } from './tabs';
+import { LayoutsListPage } from './tabs/LayoutsListPage';
```
Comment (p-fernandez) [inline apps/web/src/pages/brand/BrandPage.tsx:10]: Is there an index inside of `./tabs`? We could just have one import if so.

File: apps/web/src/pages/brand/tabs/LayoutsListPage.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    },
+    {
+      accessor: 'createdAt',
+      Header: 'Created At',
+      Cell: ({ createdAt }: any) => format(new Date(createdAt), 'dd/MM/yyyy HH:mm'),
+    },
+    {
+      accessor: 'updatedAt',
+      Header: 'Last Updated',
+      Cell: ({ updatedAt }: any) => format(new Date(updatedAt), 'dd/MM/yyyy HH:mm'),
```
Comment (p-fernandez) [inline apps/web/src/pages/brand/tabs/LayoutsListPage.tsx:74]: We can have a function inside of this file that abstracts the format functionality like:
```
const formatDate = (date: string): string => format(new Date(date), 'dd/MM/yyyy HH:mm');
```
So then we can use it like:
```
      Cell: ({ createdAt }: any) => formatDate(createdAt),
...
      Cell: ({ updatedAt }: any) => formatDate(updatedAt),
```

File: apps/web/src/pages/brand/tabs/LayoutsListPage.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+              }}
+              color={theme.colorScheme === 'dark' ? colors.B40 : colors.B80}
+            />
+          </ActionIcon>
+          <ActionIcon
+            variant="transparent"
+            onClick={(e) => {
+              e.stopPropagation();
+              onDelete(id);
+            }}
```
Comment (p-fernandez) [inline apps/web/src/pages/brand/tabs/LayoutsListPage.tsx:101]: Should we do a function `onClick` to handle this as we did with `onRowClick`?

**Review by p-fernandez — APPROVED** [review]
🌟

File: apps/web/src/pages/brand/BrandPage.tsx (hunk 1 above)
Comment (ainouzgali) [reply apps/web/src/pages/brand/BrandPage.tsx:10]: Yes! There is. Forgot to add it

File: apps/web/src/pages/brand/tabs/LayoutsListPage.tsx (hunk 2 above)
Comment (ainouzgali) [reply apps/web/src/pages/brand/tabs/LayoutsListPage.tsx:74]: Sure

File: apps/web/src/pages/brand/tabs/LayoutsListPage.tsx (hunk 3 above)
Comment (ainouzgali) [reply apps/web/src/pages/brand/tabs/LayoutsListPage.tsx:101]: For sure, I will. I just threw it in until we have the endpoint. onDelete is empty for now :)

**Pull request merged**